job/reminders/reminderSender.js

```diff
@@ -34,6 +34,14 @@ module.exports = (notificationService, incidentClient) => {
     await incidentClient.setNextReminderDate(reminder.statementId, null, client)
   }
 
+  const setNextReminderDate = async (client, reminder) => {
```

Comment:
Does this need a corresponding test?

Reply:
It's already covered - it's just extracting out a method of code that already existed